tests/cupy_tests/core_tests/test_ndarray_unary_op.py

```diff
@@ -13,7 +13,8 @@ class TestArrayBoolOp(unittest.TestCase):
 
     @testing.for_all_dtypes()
     def test_bool_empty(self, dtype):
-        assert not bool(cupy.array((), dtype=dtype))
+        with testing.assert_warns(DeprecationWarning):
+            assert not bool(cupy.array((), dtype=dtype))
```

Comment:
#4292 replaced `assertTrue` and `assertFalse` with the equivalent checks, but the intention of the original tests were probably `assert bool(...) is True` or `is False`.

https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertTrue

Reply:
But it is irrelevant to the addition of `DeprecationWarning`, which might not be backported.  I'll fix it in another PR.

Reply:
Yeah, I also thought of that.